Source/FullScreenPokemon.ts

```diff
@@ -3832,23 +3832,43 @@ module FullScreenPokemon {
 
         /**
          * Opens the context menu within the Pokedex menu for the selected Pokemon.
+         *
+         * @param settings   The selected Pokemon.
          */
         openPokemonMenuContext(settings: any): void {
+            var moves: BattleMovr.IMove[] = settings.pokemon.moves;
+            var options: any[] = [];
+            var move: IMoveSchema,
+                i: number;
+
+            for (i = 0; i < moves.length; i += 1) {
+                move = this.MathDecider.getConstant("moves")[moves[i].title];
+                if (move.partyActivate) {
+                    options.push({
+                        "text": moves[i].title.toUpperCase(),
+                        "callback": move.partyActivate(this.player, settings.pokemon)
+                    });
+                }
+            }
+
+            options.push({
+                "text": "STATS",
+                "callback": this.openPokemonMenuStats.bind(this, settings.pokemon)
+            });
+            options.push({
+                "text": "SWITCH",
+                "callback": settings.onSwitch
+            });
+            options.push({
+                "text": "CANCEL",
+                "callback": this.MenuGrapher.registerB
+            });
```

Comment:
To make this a little simpler, you can pass these all into options.push ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push))